Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml.cs

```diff
@@ -20,6 +21,15 @@ public partial class TMBadge : ContentView
     public static readonly BindableProperty BadgeShapeProperty =
         BindableProperty.Create(nameof(Shape), typeof(BadgeShape), typeof(TMBadge), BadgeShape.Rectangle, propertyChanged: OnBadgeShapePropertyChanged);
 
+    public static readonly BindableProperty BadgeContentProperty =
+        BindableProperty.Create(nameof(BadgeContent),typeof(View),typeof(TMBadge),null,propertyChanged: OnContentPropertyChanged);
+
+    public View BadgeContent
```

Comment:
Just `Content` should be enough. Can you rename?

Reply:
Its named as BadgeContent since "Content" is a property already present in ContentView and also accepts and assigns the content passed as the only content excluding the badge.